public/js/post/update_post.js

```diff
@@ -214,7 +214,7 @@ function checkPostCompleted() {
 function updateNavPost() {
     'use strict';
 
-    var navPost = $('#nav-links')[0].getElementsByTagName('li')[3].getElementsByTagName('a')[0];
+    var navPost = $('#nav-links')[0].getElementsByTagName('li')[4].getElementsByTagName('a')[0];
```

Comment:
Is it possible to get `navPost` by adding an ID to to the proper `a` element in MasterTemplate?